packages/now-routing-utils/src/superstatic.ts

```diff
@@ -75,7 +75,7 @@ export function convertHeaders(headers: NowHeader[]): Route[] {
       obj[kv.key] = kv.value;
     });
     const route: Route = {
-      src: h.source,
+      src: sourceToRegex(h.source).src,
```

Comment:
This doesn't seem right. This will allow segments in headers? We didn't spec this.

Reply:
Seems reasonable to me

Say you want to add cache headers to `/blog/:post` you don't want to have to write `/blog/([^/]+)`